integration/vite-css-dev-test.ts

```diff
@@ -36,6 +36,9 @@ test.describe("Vite CSS dev", () => {
               port: ${devPort},
               strictPort: true,
             },
+            optimizeDeps: {
+              include: ["react", "react-dom/client", "@remix-run/react"],
```

Comment:
We need this array defined so that we don't have multiple copies of these dependencies due to our project structure.